dockerfiles/functional-tests/docker-entrypoint.sh

```diff
@@ -79,6 +79,10 @@ if [[ -n $RUNNING_WORKSPACE ]]; then
 	MVN_COMMAND="${MVN_COMMAND} -Dche.workspaceName=${RUNNING_WORKSPACE}"
 fi
 
+if [[ -n $RHCHE_HOST_PROTOCOL ]]; then
```

Comment:
What about port? https://github.com/redhat-developer/rh-che/blob/master/functional-tests/pom.xml#L29 Here we are setting default value to "443". Is that correct? If not, I guess this PR should take care of this too...